### ISSUE TITLE
When you use own LN / BTC node over TOR app crahses constantly 

### ISSUE THREAD

**Issue by jlahtela** [BlueWallet/BlueWallet#3869]
I have tested usage with own LN and Bitcoin node troug TOR connectio (to Umbrel).

App seems to freeze constantly and is not usable.
First I used app own TOR and then tried to use Orbot, but it did not help the situation.

Running on Samsung Android with latest updates.

**Comment by BenGWeeks** [comment]
Possibly related to [issue 3882](https://github.com/BlueWallet/BlueWallet/issues/3882).

**Comment by jlahtela** [comment]
Maybe related to that 3882. For me it is not only in transaction app can pretty much force ant any time and then nothing works.

Also I did notice that latest transactions do not update and balances are incorrect.

For me it looks like that it tries to connect over TOR to node and it crashes. At the same time other applications connection to node works without an issue.

**Comment by BenGWeeks** [comment]
Yes, I can confirm it also doesn't update the transactions as well.

**Comment by BenGWeeks** [comment]
Would be nice it this could be fixed for Adopting Bitcoin conference.

**Comment by judesen** [comment]
+1. I have added my LNDHub network to the Lightning Settings. I can import wallets only when accessing through Tor Browser. When creating new wallets the app crashes and closes.

I am using iOS 14.7.1 on iPhone 8.

**Comment by rkfg** [comment]
Could it be the same issue as #3079?

**Comment by jlahtela** [comment]
Yes this sounds like exactly the same behavior.

**Comment by pjw65** [comment]
BlueWallet 6.2.17 freezes/crashes on Android 11 when using custom electrum server (umbrel) and/or LNDHub over internal tor.

Workaround: 
- Settings/Network/Tor disable
- Orbot VPN (add Apps/BlueWallet)
